### ISSUE TITLE
Popover not showing when clicking

### ISSUE THREAD

**Issue by Paliverse** [iSapozhnik/Popover#9]
I've followed all steps and not working at all, the view is added to the menu bar, but not showing the ViewController when clicking on it. Is it broken for macOS Monterey?

**Comment by iSapozhnik** [comment]
Hmm.. I don't have Monterey installed on my machine yet to check it. Either you can check what is the problem or we can keep this issue open until I have Monterey.

**Comment by Paliverse** [comment]
Yeah it’s weird, Code might need an update for Monterey even tho Xcode isn’t saying there’s any errors or warnings. I decided to just go with how popover is implemented as default. But one thing to note is that when adding an image or a view, it is added to the menu bar, just when clicking, the actual VC isn’t popping up. I’ll see if I can figure it out if I download your source and update you on it
